great_expectations/rule_based_profiler/config/base.py

```diff
@@ -303,6 +303,34 @@ def get_config_class(cls) -> Type["RuleBasedProfilerConfig"]:  # noqa: F821
     def get_schema_class(cls) -> Type["RuleBasedProfilerConfigSchema"]:  # noqa: F821
         return RuleBasedProfilerConfigSchema
 
+    # noinspection PyUnusedLocal,PyUnresolvedReferences
+    @classmethod
+    def resolve_config_using_acceptable_arguments(
```

Comment:
@cdkini I do not see `resolve_config_using_acceptable_arguments()` being used anywhere (other than in tests).  Is this here for a next PR?  Thanks!

Reply:
yes it is to come in the final PR.